### PR TITLE
Created separate Installation instructions

### PR DESCRIPTION
**Pull request by Rohan-G**
Added separate installation instructions for Linux, Windows and MacOS.
Ran the commands on Linux and installation works fine (even ran with one of the samples).

### CODE REVIEW

**Comment by Rohan-G** [comment]
@Udayraj123 could you review this and let me know if any changes are needed?

File: README.md [hunk 1]
```diff
@@ -103,57 +103,122 @@ Get a CSV sheet containing the detected responses and evaluated scores:
 
 **Operating system:** OSX or Linux is recommended although windows is also supported currently.
 
-### 1. Install global dependencies
+## 1. Installation procedures
```
Comment (Udayraj123) [inline README.md:106]: Let's just use `Installation`

File: README.md [hunk 2]
```diff
@@ -103,57 +103,122 @@ Get a CSV sheet containing the detected responses and evaluated scores:
...
+**Installing missing libraries(if any):** 
 
 On a fresh computer, some of the libraries may get missing in event after a successful pip install. Install them using following commands[(ref)](https://www.pyimagesearch.com/2018/05/28/ubuntu-18-04-how-to-install-opencv/):
 
 ```bash
-sudo apt-get install -y build-essential cmake unzip pkg-config
-sudo apt-get install -y libjpeg-dev libpng-dev libtiff-dev
-sudo apt-get install -y libavcodec-dev libavformat-dev libswscale-dev libv4l-dev
-sudo apt-get install -y libatlas-base-dev gfortran
+sudo apt-get install -y build-essential cmake unzip pkg-config libjpeg-dev libpng-dev libtiff-dev libavcodec-dev libavformat-dev libswscale-dev libv4l-dev libatlas-base-dev gfortran
```
Comment (Udayraj123) [inline README.md:126]: No keeping it separate is better as facing error in either of the packages would be easier to debug in separate steps

File: README.md [hunk 3]
```diff
@@ -103,57 +103,122 @@ Get a CSV sheet containing the detected responses and evaluated scores:
...
 ```
 
-### 2. Install project dependencies
-
+#### 2. Install project dependencies
 <!-- ![imutils 0.5.2](https://img.shields.io/badge/imutils-0.5.2-blue.svg)  -->
+![matplotlib 3.0.2](https://img.shields.io/badge/matplotlib-3.0.2-blue.svg) ![pandas 0.24.0](https://img.shields.io/badge/pandas-0.24.0-blue.svg) ![numpy 1.16.0](https://img.shields.io/badge/numpy-1.16.0-blue.svg) ![deepmerge 0.1.0](https://img.shields.io/badge/deepmerge-0.1.0-blue.svg) ![dotmap 1.3.17](https://img.shields.io/badge/dotmap-1.3.17-blue.svg)
```
Comment (Udayraj123) [inline README.md:131]: Let's remove these badges as they might not be in sync with requirements.txt

File: README.md [hunk 4]
```diff
@@ -103,57 +103,122 @@ Get a CSV sheet containing the detected responses and evaluated scores:
...
+
+##### Clone the repo
+```bash
+# Shallow clone - takes latest code with minimal size
+git clone https://github.com/Udayraj123/OMRChecker --depth=1
```
Comment (Udayraj123) [inline README.md:136]: can you take the latest pull over this? I have made changes in the line

File: README.md [hunk 5]
```diff
@@ -103,57 +103,122 @@ Get a CSV sheet containing the detected responses and evaluated scores:
...
+
+#### Install pip requirements 
+
+```bash
+# cd OMRChecker/
+python3 -m pip install --user -r requirements.txt
+```
+**Friendly Note:** If you face a distutils error in pip, use `--ignore-installed` flag in above command.
+
+### **<u>Windows Users</u>**
```
Comment (Udayraj123) [inline README.md:149]: Let's experiment creating foldable markdown in Readme for installation instructions. Can save space.
like this one https://github.com/Udayraj123/OMRChecker/pull/57/files

**Comment by Udayraj123** [comment]
@Rohan-G can you please take latest pull again?

**Pull request closed without merging**